info.yml

```diff
@@ -32,52 +32,61 @@ configuration:
                      for a platform, just include it in the the deny_platforms list. Valid values 
                      are Windows, Mac and Linux."    
     
-    viewer_extensions: 
+    valid_extensions:
         type: list
         values: {type: str}
-        default_value: [exr, cin, dpx]
-        description: "A list of file extensions to send to the viewer application.
+        default_value: []
+        allows_empty: True
+        description: "A list of file extensions to consider as valid in hook_get_published_file.
+                     The order of the extensions can be useful to determine which published file
+                     to pick in case there are more than one.
+                     An example default implementation is {self}/shotgun_get_valid_published_file.py
                      Do not include the period character. Example: `[exr, cin, dpx]`"
-    
-    viewer_path_windows:       
+
+    app_path_windows:
         type: str
-        default_value: "C:\\Program Files\\RV\\RV64.exe"
-        description: A path to an image sequence viewer for windows. Any path with 
-                     an extension defined in the viewer_extensions list will be 
-                     dispatched here.
-    viewer_path_linux:       
+        default_value: ""
+        description: "A path to an app for Windows. It needs to
+                      be defined to use the hook shotgun_launch_app."
+    app_path_linux:
         type: str
-        default_value: "rv"
-        description: A path to an image sequence viewer for linux. Any path with 
-                     an extension defined in the viewer_extensions list will be 
-                     dispatched here.
-    viewer_path_mac:       
+        default_value: ""
+        description: "A path to an app for Linux. It needs to
+                      be defined to use the hook shotgun_launch_app."
+    app_path_mac:
         type: str
-        default_value: "/Applications/RV64.app"
-        description: A path to an image sequence viewer for mac. Any path with 
-                     an extension defined in the viewer_extensions list will be 
-                     dispatched here.
```

Comment:
by removing the references to "viewer" it's less obvious how a single app_path would be useful when dealing with lots of different filetypes. It would be nice to add something that states this could typically be used for specifying a path to a viewer.

Reply:
You could add multiple configurations of the same app to do different launches... Launch in viewer, launch in photoshop, etc, provided that the studios can give one single path to an app for all users. But you are right about adding details, and I was also thinking about adding more info in the README? But normally SG does not do that and since at some point I'd like to submit this to SG I'm not sure where to put that information..